const isLoggedIn = async (req, res, next) => {
	try {
	} catch (error) {}

	next();
};
